.buildkite/pipeline.yml

```diff
@@ -101,6 +101,32 @@ steps:
                     - BUILDKITE_PULL_REQUEST_BASE_BRANCH=${BUILDKITE_PULL_REQUEST_BASE_BRANCH}
                     - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
 
+  - label: "Trigger x-pack/metricbeat"
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only origin/${GITHUB_PR_TARGET_BRANCH}...HEAD"
+          watch:
+            - path:
+                - x-pack/metricbeat/
+                - x-pack/libbeat/common/aws
+                - .buildkite/x-pack/pipeline.xpack.metricbeat.yml
+                - .buildkite/scripts
```

Comment:
taken form the discussion today, let include also the hooks

Reply:
thanks! fixed in 7066d4cd56c0adff9402e707d2df33c3f6c1ea04